pandas/tests/arithmetic/test_datetime64.py

```diff
@@ -1019,6 +1019,7 @@ def test_dt64arr_add_timestamp_raises(self, box_with_array):
             np.array([2.0, 3.0]),
             # GH#13078 datetime +/- Period is invalid
             pd.Period("2011-01-01", freq="D"),
+            time(1, 2, 3),
```

Comment:
can you add a comment on the previous line pointing to the appropriate issue (like the comment on L1020 for Period)